prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -1,54 +1,87 @@
-* Display Interstitial via `InterstitialAdUnit`
-* Video Interstitial via `VideoInterstitialAdUnit`
-* Rewarded Video via `RewardedVideoAdUnit`
-* Native Styles via `NativeRequest`
+Before implementing the instructions in this guide, you need to ensure that you have correctly initialized the Prebid Mobile SDK in your app. The initialization method has the following pattern:
 
-The basic integration steps for these ad units you can find on the page for integration using [Original API](/prebid-mobile/pbm-api/android/android-sdk-integration-gam-original-api.html). The difference is that you should use  the `fetchDemand` function with the following signature:
+```java
+Prebid.init(context, accountID, adServer, host);
+```
 
-```kotlin
+To use the above code snippet, you will need to [setup a Prebid server.](/prebid-mobile/prebid-mobile-getting-started.html#set-up-prebid-server) You can either register with a [Prebid.org member that hosts Prebid Server](https://prebid.org/managed-services/) or [setup your own Prebid server.](/prebid-server/hosting/pbs-hosting.html)
+
+* `context`: This parameter represents the Context of the Android application. It is typically obtained using `getApplicationContext()` or `this` (if called from within an Activity). The Context is essential for the SDK to interact with the Android environment.
+
+* `adUnitId`: This is the value of the adunit name in the ad server. e.g. for GAM, it's the adunit name. Other ad servers may call this something different, but the idea is that the buyers want to know which slot in the app is up for auction for targeting and reporting. An example would be "/home/upper-mobile-banner".
+
+* `accountId`: The accountId is used to define the accountID and the "account settings" used for this app in Prebid Server. These allow your Prebid Server to lookup account details and attributes like timeout and price granularity.
+
+* `adServer`: This parameter specifies the ad server you are using. It is an enum value of the type `Prebid.AdServer`. The possible values include `Prebid.AdServer.DFP` for Google Ad Manager and `Prebid.AdServer.MOPub` for MoPub.
+
+* `host`: If you are using a managed Prebid server service, the SDK is coded with enums containing the URLs for those services, for example `Prebid.Host.RUBICON` and `Prebid.Host.APPNEXUS`. Other Prebid Server managed services exist that don't have enum values. You can work with them by entering the appropriate URL. If you are using a self-hosted server, you should enter the URL to your server.
+
+For example:
+
+```java
+Prebid.init(getApplicationContext(), adUnits, "INSERT-ACCOUNT-ID-HERE", Prebid.AdServer.DFP, "https://mypbs.example.org");
+```
+
+## Define first party data and set up adUnits
+
+When using a custom or 3rd-party ad server, you must first get the targeting keys using the `fetchDemand` method. This function provides the bidder key/values (targeting keys). You can then pass these targeting keys to the ad server of your choice.
```

Comment:
I added the First Party Data part, but the real problem with the structure of this section is that you have to set up adunits BEFORE calling `fetchDemand`. 

Please restructure this page so that the flow makes sense:
- initialize
- set up the adunits and first party data
- call fetchDemand
- render